packages/medusa-js/src/resources/admin/batch-jobs.ts

```diff
@@ -24,7 +25,7 @@ class AdminBatchJobsResource extends BaseResource {
     let path = `/admin/batch-jobs`
 
     if (query) {
-      const queryString = qs.stringify(query)
+      const queryString = qs.stringify(stringifyNullProperties(query))
```

Comment:
qs supports `{skipNulls: true}` config, would that be sufficient?

Reply:
no, `skipNull` remove the key from the query params when it is null, and we want the opposite  :)